src/index.py

```diff
@@ -28,6 +28,15 @@ def handler(event, lambda_context):
     dt_token = os.environ.get('DYNATRACE_API_KEY')
     verify_SSL = os.environ.get('VERIFY_SSL', 'false') == 'true'
 
+    try:
+        with open('version.txt') as versionFile:
+            version = versionFile.readline()
+    except Exception as e:
+        version = "?"
+        log_error_with_stacktrace(e, "Couldn't read stack version", "version-reading-exception")
```

Comment:
I think this is not error, this should be INFO/WARN as this is expected during local development etc. 
Just INFO/WARN that cannot read version from file + exception.msg 
Thanks